packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -9,14 +9,14 @@ export interface SectionDefinition {
 export interface FieldDefinition {
   id: string;
   type: string;
-  label: string;
+  label: string | null;
```

Comment:
We can also use `label?: string` instead of using `|null`. Both representations denotes that this value can be null.

Just a suggestion.
Thanks!

Reply:
Optional and "possibly null" are distinct. Optional is more or less the same as `| undefined`. In this case, the object will always have the `label` property, but it will sometimes be null-valued.